woocommerce/woocommerce-android
# Fix/update fastlane

This PR updates `Fastlane` to get rid of the `mini_magick` security alert.

To test:
1. Run `bundle install`.
2. Run one of the release lanes (for example: build_pre_releases) and verify that it works.

Update release notes:
